generate_db.py

```diff
@@ -3,11 +3,11 @@
 import sys
 import os
 
-language = sys.argv[1]
+language = sys.argv[1] # the expected input is enwiki
```

Comment:
is the expected input en or enwiki or en.wikipedia?

Reply:
The expected input is enwiki but I think en would be better(?) as the behavior right now is a little confusing.